airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/input_config.py

```diff
@@ -0,0 +1,73 @@
+#
```

Comment:
I think I don't fully understand what is the purpose of this config and the scripts around it. How do you see the new DX flow?
I think running both - unit and international tests should be easy and use minimal or 0 external toolsets, so `pytest unit_tests` or `pytest integration_tests` should be enough.
SAT is a bit different in this case because it doesn't execute directly but interacts with the docker image via Airbyte protocol.

Reply:
the only thing SAT needs is the path to the config file, all paths inside this file will be relative to the config file location. It is not necessary to mean that it is located in the connector source folder or we have a connector source at all. SAT written with the possibility to test any docker image that follows Airbyte protocol, executing as much generic tests as needed to verify that connector follows Aibyty protocol, conventions, and guarantees/promises.